dockerfiles/header_sync/Dockerfile

```diff
@@ -0,0 +1,41 @@
+FROM golang:alpine as builder
+
+RUN apk --update --no-cache add g++ git linux-headers
+
+ENV GO111MODULE on
+
+WORKDIR /vulcanizedb
```

Comment:
Should probably copy the project code *after* building goose, so we can cache that layer :)